### PR TITLE
Update bug reporting link to Git Issues. Bug #5647 

### PR DESCRIPTION
**Pull request by ankiiitraj**
Changes made to the links of reporting a bug from "https://bugzilla.mozilla.org/form.mdn" to "https://github.com/mozilla/kuma/issues/new" .
Changes made to the text Label from "Report a bug" to "Report an issue".

### CODE REVIEW

**Comment by ankiiitraj** [comment]
@schalkneethling Here's the Pull Request, I was unable to assign you as a Reviewer.

**Comment by ankiiitraj** [comment]
> @schalkneethling Here's the Pull Request, I was unable to assign you as a Reviewer.

Am I done with this Issue and move to other issues or I have something else to do here. Please Guide me.
Thanks

**Comment by schalkneethling** [comment]
> > @schalkneethling Here's the Pull Request, I was unable to assign you as a Reviewer.
> 
> Am I done with this Issue and move to other issues or I have something else to do here. Please Guide me.
> Thanks

I am going to review this today. If all is good I will approve and merge. Will let you know @ankiiitraj

File: jinja2/500.html [hunk 1]
```diff
@@ -32,7 +32,7 @@ <h1 class="page-title">{{_('Internal Server Error')}}</h1>
       {%- endif %}
     {% endif %}
     <p>If you have additional information how to reproduce this problem, please
-    <a href="https://bugzilla.mozilla.org/form.mdn" rel="nofollow">file a bug.</a>
+    <a href="https://github.com/mozilla/kuma/issues/new" rel="nofollow">file a bug.</a>
```
Comment (schalkneethling) [inline jinja2/500.html]:
```suggestion
    <a href="https://github.com/mozilla/kuma/issues/new" rel="nofollow">file an issue.</a>
```

File: kuma/javascript/src/header/__snapshots__/header.test.js.snap [hunk 2]
```diff
@@ -405,11 +405,11 @@ exports[`Header snapshot 1`] = `
             role="menuitem"
           >
             <a
-              href="https://bugzilla.mozilla.org/form.mdn"
+              href="https://github.com/mozilla/kuma/issues/new"
               rel="noopener noreferrer"
               target="_blank"
             >
-              Report a bug
+              Report an issue
```
Comment (schalkneethling) [inline kuma/javascript/src/header/__snapshots__/header.test.js.snap]: For future reference, these `.snap` files are auto generated by [Jest](https://jestjs.io/) so, you do not(should not) update them manually. The way to do it is to run the following from the terminal(https://jestjs.io/docs/en/cli#updatesnapshot):

```
npm t -- --u
```

**Comment by schalkneethling** [comment]
@ankiiitraj We are getting a Github issue template in place, and as soon as that is ready, I will merge this. Thanks again.

**Comment by ankiiitraj** [comment]
I am prety much pumped with it, looking forward for more such adventures. Thanks.

**Comment by callahad** [comment]
This is great, thank you for the contribution, @ankiiitraj!

**Comment by callahad** [comment]
Manually squashed and merged in 56392b7af48680dcb4e9ac2f4b85efd018569fb7 to preserve authorship.

**Comment by callahad** [comment]
Sorry for all the opening/closing of this issue... we're experimenting with a project management tool and it's inadvertently toggling this issues. :grin:

**Pull request closed without merging**